efficientdet/keras/inspector_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+r"""Tests for model inspect tool."""
+import os
+import shutil
+import tempfile
+import subprocess
+
+from absl import logging
+import tensorflow as tf
+
+
+class ModelInspectTest(tf.test.TestCase):
+  """Model inspect tests."""
+
+  def setUp(self):
+    super().setUp()
+    sys_tempdir = tempfile.mkdtemp()
+    self.tempdir = os.path.join(sys_tempdir, '_inspect_test')
+    os.mkdir(self.tempdir)
+
+    self.savedmodel_dir = os.path.join(self.tempdir, 'savedmodel')
+    if os.path.exists(self.savedmodel_dir):
+      shutil.rmtree(self.savedmodel_dir)
+
+  def tearDown(self):
+    super().tearDown()
+    shutil.rmtree(self.tempdir)
+
+  def test_dry(self):
+    result = subprocess.run(
+        ['python3', 'keras/inspector.py', '--mode=dry', '--ckpt_path=_'])
```

Comment:
Maybe we we could make it simpler by setting flags? such as:

from keras import inspector
from absl import flags
FLAGS = flags.FLAGS


def test_dry(self):
  FLAGS.mode = 'dry'
  FLAGS.ckpt_path = '_'
  inspector.main(None)
  self.assert(...) 

What do you think?

Reply:
FLAGS shares values in different test cases which breaks other test cases.
For example: ```infer_test``` gets saved_model_dir which is saved in ```export_test```.

Reply:
you can use flagsaver. I will merge this PR, and send you a new PR to show the usage.